ServerCore/Pages/Teams/AutoTeam.razor

```diff
@@ -0,0 +1,22 @@
+@using ServerCore.DataModel;
+
+<p>I&apos;m a...</p>
+<EditForm Model="this" OnValidSubmit="OnSubmit">
+    <DataAnnotationsValidator />
+    <ValidationSummary />
+
+    <InputRadioGroup Name="PlayerTypeSelector" @bind-Value="PlayerType">
```

Comment:
There was discussion in a recent PD meeting about perhaps altering this to incorporate how serious about the event the player is. Is that something we should incorporate this year or no?

Reply:
Done